src/backend/marsha/core/lti/__init__.py

```diff
@@ -20,6 +19,8 @@ class LTI:
     It provides properties and methods to inspect the launch request.
     """
 
+    _consumer_site = None
```

Comment:
to be removed as you made it an instance attribute?

Reply:
Yes